### DISCUSSION TITLE
With the new installer and forked development - is it now possible to have two co-existing installations?

### DISCUSSION THREAD

**Discussion by Annolago** [flybywiresim/aircraft#4309] [Q&A]
Hi
Just a quick question. Since the FWB A32 is now a separate plane type I was wondering if it would be possible to have multiple, or at least two, concurrent installations at the same time. One would be either the stable  or development versions and the other the Experimental version. Now my thinking is that the stable/development version overwrites the build-in A320Neo in FS2020, but the experimental seems to create a separate subfolder and is listed as a completely new plane in game - or have I missed something?
If not possible then maybe something to consider as a future idea. 
Oh - and again again thanks for all the hard work you guys are investing in this project. You really should have a tip jar for when ever it will be possible to meet up again and brake bread together.

**Comment by ExampleWasTaken** [comment]
As far as I know it is not possible to have two versions of the plane in the community folder at the moment.

And for the tip jar you mentioned: https://opencollective.com/flybywire
